src/coreclr/vm/jitinterface.cpp

```diff
@@ -9545,6 +9536,22 @@ uint32_t CEEInfo::getLoongArch64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE c
                         size = getLoongArch64PassStructInRegisterFlags((CORINFO_CLASS_HANDLE)pNativeFieldDescs->GetNestedNativeMethodTable());
                         return size;
                     }
+                    else if (pNativeFieldDescs->GetCategory() == NativeFieldCategory::FLOAT)
+                    {
+                        if (pNativeFieldDescs->NativeSize() == 4)
+                        {
+                            size = STRUCT_FLOAT_FIELD_ONLY_ONE;
+                        }
+                        else if (pNativeFieldDescs->NativeSize() == 8)
+                        {
+                            size = STRUCT_FLOAT_FIELD_ONLY_ONE;
+                        }
+                        else
```

Comment:
I have a question:
what case is that `pNativeFieldDescs->NativeSize() == 1` while `pNativeFieldDescs->GetCategory() == NativeFieldCategory::FLOAT` ?